Source/Core/VideoCommon/PerformanceTracker.h

```diff
@@ -89,16 +78,15 @@ class PerformanceTracker
   // Queue + Running Total used to calculate average dt
   DT m_dt_total = DT::zero();
   std::array<DT, MAX_DT_QUEUE_SIZE> m_dt_queue;
-  std::size_t m_dt_queue_begin = 0;
-  std::size_t m_dt_queue_end = 0;
+  u64 m_dt_queue_begin = 0, m_dt_queue_end = 0;
 
   // Average rate/time throughout the window
   DT m_dt_avg = DT::zero();  // Uses Moving Average
   double m_hz_avg = 0.0;     // Uses Moving Average + Euler Average
 
   // Used to initialize this on demand instead of on every Count()
-  mutable std::optional<DT> m_dt_std = std::nullopt;
+  mutable std::optional<DT> m_dt_std;
```

Comment:
Any reason you removed this? It doesn't change behavior.

Reply:
i dont remember but ill just add it back.